kernel/arch/dreamcast/include/dc/pvr.h

```diff
@@ -2131,6 +2133,52 @@ void pvr_dma_init(void);
 /** \brief  Shut down PVR DMA. */
 void pvr_dma_shutdown(void);
 
+/** \brief   Copy a block of memory to VRAM
+    \ingroup store_queues
+
+    This function is similar to sq_cpy(), but it has been
+    optimized for writing to a destination residing within VRAM.
+
+    \warning
+    This function cannot be used at the same time as a PVR DMA transfer.
+
+    The dest pointer must be at least 32-byte aligned and reside 
+    in video memory, the src pointer must be at least 8-byte aligned, 
+    and n must be a multiple of 32.
+
+    \param  dest            The address to copy to (32-byte aligned).
+    \param  src             The address to copy from (32-bit (8-byte) aligned).
+    \param  n               The number of bytes to copy (multiple of 32).
+    \param  type            The type of SQ/DMA transfer to do (see list of modes).
+    \return                 The original value of dest.
+
+    \sa sq_cpy()
+*/
+void *pvr_sq_load(void *dest, const void *src, size_t n, int type);
+
+/** \brief   Set a block of PVR memory to a 16-bit value.
+    \ingroup store_queues
+
+    This function is similar to sq_set16(), but it has been
+    optimized for writing to a destination residing within VRAM.
+
+    \warning
+    This function cannot be used at the same time as a PVR DMA transfer.
+    
+    The dest pointer must be at least 32-byte aligned and reside in video 
+    memory, n must be a multiple of 32 and only the low 16-bits are used 
+    from c.
+
+    \param  dest            The address to begin setting at (32-byte aligned).
+    \param  c               The value to set (in the low 16-bits).
+    \param  n               The number of bytes to set (multiple of 32).
+    \param  type            The type of SQ/DMA transfer to do (see list of modes).
+    \return                 The original value of dest.
+
+    \sa sq_set(), sq_set16(), sq_set32()
```

Comment:
Im curious if these will generate Doxygen warnings not that they are in a different file (sq.c)?

Reply:
@DC-SWAT You can check if this generates warnings by navigating to kos/doc folder and run: doxygen Doxyfile